DataGrid - Getting Started: Add ASP.NET Core Overhaul (#8900)
DevExpress/devextreme-documentation: concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/70 Configure Master-Detail Interface.md

#include common-tutorialbutton-named with { url: "/Documentation/Guide/UI_Components/DataGrid/Master-Detail_Interface/", name: "DataGrid - Master-Detail Interface" }

> DataGrid supports master-detail data presentation. You can display detail data in expandable sections under master rows. To configure a master-detail data grid, set **masterDetail**.[enabled](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/enabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#enabled') to **true** and specify a [template](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/template.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#template'). This tutorial configures a template that displays employee images using file paths stored in the component data source:

Reviewer: ```suggestion
DataGrid supports master-detail data presentation. You can display detail data in expandable sections below master rows. To configure master-detail mode, set **masterDetail**.[enabled](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/enabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#enabled') to **true** and specify a [template](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/template.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#template'). This tutorial configures a template that displays employee images from file paths stored in the component data source:
```

> DataGrid supports master-detail interfaces. These allow you to display additional details in expandable sections under data rows. To configure master-detail interfaces, set **masterDetail**.[enabled](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/enabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#enabled') to **true** and specify a [template](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/template.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#template'). This tutorial configures a template that displays employee images using file paths stored in the component data source:

Reviewer 2: ```suggestion
DataGrid supports master-detail data presentation. You can display detail data in expandable sections under master rows. To configure a master-detail data grid, set **masterDetail**.[enabled](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/enabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#enabled') to **true** and specify a [template](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/masterDetail/template.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/masterDetail/#template'). This tutorial configures a template that displays employee images using file paths stored in the component data source:
```
